packages/embeds/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@orbiting/backend-modules-embeds",
+  "version": "0.0.1",
+  "private": true,
+  "description": "query external resources",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/orbiting/backends.git"
+  },
+  "author": "Patrick Recher <patrick.recher@republik.ch>",
+  "license": "AGPL-3.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "bugs": {
+    "url": "https://github.com/orbiting/backends/issues"
+  },
+  "homepage": "https://github.com/orbiting/backends#readme",
+  "dependencies": {
```

Comment:
I find it comforting and useful when adding all external package dependencies (`... require(...)`) to `package.json`. (`@orbiting/...` not necessary, there is always one version which yarn handles.) Will help when exporting either module as a standalone version. Until then it comes in super-handy when upgrading packages (e.g. `yarn upgrade-interactive --latest`) and works swiftly as a documentation.

Reply:
I just forgot, thanks for spotting it.